api/test/integration/test_rbac_white_agents_endpoints.tavern.yaml

```diff
@@ -1420,6 +1420,41 @@ stages:
       json:
         <<: *permission_denied
 
+---
+test_name: PUT /groups/{group_id}/restart
+
+stages:
+
+    # PUT /groups/default/restart
+  - name: Restart all agents from group default (Partially allowed)
+    delay_before: 20
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/{version:s}/groups/default/restart"
+      method: PUT
+      headers:
+        Authorization: "Bearer {test_login_token}"
+        content-type: "application/xml"
+    response:
+      status_code: 200
+      json:
+        data:
+          affected_items:
+            - "004"
+          total_affected_items: 1
+          failed_items:
+            - error:
```

Comment:
I would remove all checks except code 4000 to avoid excess compromise with the messages.